apps/rush-lib/src/cli/actions/CheckAction.ts

```diff
@@ -6,10 +6,11 @@ import { RushCommandLineParser } from '../RushCommandLineParser';
 import { BaseRushAction } from './BaseRushAction';
 import { VersionMismatchFinder } from '../../logic/versionMismatch/VersionMismatchFinder';
 import { Variants } from '../../api/Variants';
-import { CommandLineStringParameter } from '@microsoft/ts-command-line';
+import { CommandLineStringParameter, CommandLineFlagParameter } from '@microsoft/ts-command-line';
```

Comment:
```suggestion
import {
  CommandLineStringParameter,
  CommandLineFlagParameter
} from '@microsoft/ts-command-line';
```

Reply:
This should also be moved above the local imports, with a blank line between dependency reference and local references.